### ISSUE TITLE
Add upload and download button

### ISSUE THREAD

**Issue by the-wulv** [Skalman/UglifyJS-online#9]
Currently, the only way to minify a Javascript file with this tool is to open it, copy everything and paste them in the website. It would be much more easy if there existed an **upload button** for uploading Javascript files.

Also, to store the minified code, copy/pasting is required again, so a **download button** that downloads the minified Javascript file would also make things easier.

**Comment by Skalman** [comment]
@the-wulv, I agree, this seems like a good addition. Feel free send a pull request with this feature.
